content/influxdb/cloud-dedicated/reference/sql/functions/time-and-date.md

```diff
@@ -360,6 +361,276 @@ GROUP BY _time, room
 {{% /expand %}}
 {{< /expand-wrapper >}}
 
+## date_bin_wallclock
+
+Calculates time intervals using the timezone of a specified time value and
+returns the start of the interval nearest to the specified timestamp.
+Use `date_bin_wallclock` to downsample time series data by grouping rows into
+time-based "bins" or "windows" that are based off "wall clock" times in a
+specific timezone and applying an aggregate or selector function to each window.
+
+### Time zone shifts
+
+Many regions use time zone shifts (such as Daylight Saving Time (DST) and British
+Summer Time (BST)). If a wall clock time bin starts at a time that does not
+exist in the specified time zone, the timestamp is adjusted to the time that is
+the same offset from the start of the day in that time zone.
+
+If a wall clock time represents an ambiguous time in the region then the
+behavior depends on the size of the specified interval. If the interval is
+larger than the difference between the two possible timestamps, then the earlier
+timestamp is used. Otherwise, the function uses the timestamp that matches the
+UTC offset of the input timestamp.
+
+```sql
+date_bin_wallclock(interval, expression[, origin])
```

Comment:
```suggestion
date_bin_wallclock(interval, expression[, origin_timestamp])
```

The arguments defined below uses `origin_timestamp`. To keep wording consistent, maybe changing the word to `origin_timestamp` here, or changing the word to `origin` below?